bootstap 5 update (#976)
shopware/docs: guides/plugins/plugins/storefront/using-a-modal-window.md

## Create a modal using AjaxModalPlugin

> When setting a `data-url` in addition to `data-bs-toggle="modal"` shopware automatically uses the `PseudoModalUtil` and the pseudo modal template from the `base.html.twig` to render a modal:

Reviewer: ```suggestion
When setting `data-ajax-modal="true"` together with `data-url` shopware automatically uses the `PseudoModalUtil` and the pseudo modal template from the `base.html.twig` to render a modal:
```

We actually changed this in 6.5.0 to avoid interfering with the default Bootstrap modal opening mechanism, and also separate it more clearly as a shopware JS-plugin.

> Deprecated selector `[data-bs-toggle="modal"][data-url]` to initialize the  AjaxModal plugin. Use selector `[data-ajax-modal][data-url]` instead.

The current syntax still works until major version 6.6.0, but it produces a JS error in the console and `data-ajax-modal` is the recommended way.

Related changelog and upgrade notes: https://github.com/shopware/platform/blob/trunk/changelog/release-6-5-0-0/2023-01-04-fix-console-error-on-ajax-modal.md

{% block base_main_inner %}
    <!-- This uses `AjaxModalPlugin` -->
    <button class="btn"

>             data-bs-toggle="modal"

Reviewer: ```suggestion
            data-ajax-modal="true"
```
